internal/plugin/service/organization/organization_application_user_data_source.go

```diff
@@ -67,22 +67,22 @@ func (r *organizationApplicationUserDataSource) Schema(
 	resp *datasource.SchemaResponse,
 ) {
 	resp.Schema = schema.Schema{
-		Description: "Retrieves information about an organization application user from Aiven.",
+		Description: "Gets information about an application user.",
 		Attributes: map[string]schema.Attribute{
 			"organization_id": schema.StringAttribute{
-				Description: "Identifier of the organization the application user belongs to.",
+				Description: "The ID of the organization the application user belongs to.",
 				Required:    true,
 			},
 			"user_id": schema.StringAttribute{
-				Description: "Identifier of the organization application user.",
+				Description: "The ID of the application user.",
 				Required:    true,
 			},
 			"name": schema.StringAttribute{
-				Description: "Name of the organization application user.",
+				Description: "Name of the application user.",
 				Computed:    true,
 			},
 			"email": schema.StringAttribute{
-				Description: "Email of the organization application user.",
+				Description: "The auto-generated email address of the application user.",
```

Comment:
I took most of the descriptions from the Aiven Public API docs. Perhaps, this should also be adjusted in them, too?

Reply:
Good point. The API docs are also being (slowly) updated/improved, so we'll get to this eventually...